include/uv.h

```diff
@@ -1096,6 +1104,9 @@ struct uv_fs_s {
   void* ptr;
   const char* path;
   uv_stat_t statbuf;  /* Stores the result of uv_fs_stat() and uv_fs_fstat(). */
+  const uv_dir_t* dir; /* Stores the result of uv_fs_opendir() */
+  uv_dirent_t* dirents;
+  size_t nentries;
```

Comment:
I guess it's possible without that addition, say with an own struct right? That of course breaks ABI compatibility.

Reply:
Can you add methods similar to `uv_fs_get_statbuf` etc. from 0d6525acae3b9 ?